src/js/components/Brick.js

```diff
@@ -0,0 +1,47 @@
+// (C) Copyright 2014-2015 Hewlett Packard Enterprise Development LP
+
+import React, { Component } from 'react';
+import Anchor from './Anchor';
+
+const CLASS_ROOT = "brick";
+
+export default class Brick extends Component {
+  render() {
+    var classes = [CLASS_ROOT];
+    classes.push('brick--' + this.props.width + '-' + this.props.height);
+
+    if (this.props.colorIndex) {
+      classes.push('background-color-index-' + this.props.colorIndex);
+    }
+    if (this.props.className) {
+      classes.push(this.props.className);
+    }
+
+    var label = <div className="brick--label"><span>{this.props.label}</span></div>;
```

Comment:
these too lines seems to long, can you break it into multiple lines? i need to investigate how to add that to our linters. our code itself has a bunch of issues like this.

<img width="967" alt="screen shot 2016-01-14 at 12 16 54 am" src="https://cloud.githubusercontent.com/assets/1207250/12314938/3f0adbf8-ba54-11e5-8f87-33d259b398f9.png">

The idea is to stay behind the line